.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+  
+jobs:
+  build-docs:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
+      # Specify service dependencies here if necessary
+    environment:
+      BRANCH: gh-pages 
+      REPO: git@github.com:hyphacoop/handbook.git
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run:
+          name: "Install GitBook"
+          command: |
+            npm install gitbook-cli
+
+      - run:
+          name: "Clone gh-pages"
+          command: |
+            git clone --branch $BRANCH $REPO ~/$BRANCH
```

Comment:
Can we cite the tutorial we derived this from? Finding different instructions, and that's a-ok, but would be nice to know where this approach came from :)